### ISSUE TITLE
Python syntax highlighting not working

### ISSUE THREAD

**Issue by ciarancourtney** [vitalibo/bitbake-intellij-plugin#2]
For example the bb python function below is treated as shell:

```
python do_build() {
    print("build")
}
```

**Comment by fbezdeka** [comment]
I tried this plugin today and I had to configure a python interpreter (project settings) first.

Highlighting works, but I'm facing some indention and python errors. Keeping the first line of the function empty solves them.

Reports "unexpected indent":
```python
python test() {
    a = "as"
    if a in ["asdf"]:
        print("Match")
}
```

Works:
```python
python test() {
    
    a = "as"
    if a in ["asdf"]:
        print("Match")
}
```

**Comment by fbezdeka** [comment]
Closing as this is nearly two years old now. To make the python highlighting work it might be necessary to configure a python interpreter in the project settings. The mentioned problems above should be fixed in version >= 1.4.
